blockly-ws-search/package.json

```diff
@@ -45,7 +46,11 @@
     "webpack-dev-server": "^3.10.3"
   },
   "peerDependencies": {
-    "blockly": "^3.20200123.1"
+    "blockly": ">=3.20200123.0"
```

Comment:
we probably want to add an upper bound, but the explicit >= (and then < to be added) could be clearer than using "^" we had before.